Libraries/Storage/AsyncStorage.js

```diff
@@ -333,19 +332,19 @@ if (!RCTAsyncStorage.multiMerge) {
   delete AsyncStorage.multiMerge;
 }
 
-function convertErrors(errs) {
+function convertErrors(errs): ?Array<?Error> {
   if (!errs) {
     return null;
   }
   return (Array.isArray(errs) ? errs : [errs]).map(e => convertError(e));
 }
 
-function convertError(error) {
+function convertError(error): ?Error {
   if (!error) {
     return null;
   }
-  const out = new Error(error.message);
-  out.key = error.key; // flow doesn't like this :(
+  const out: Object = new Error(error.message);
+  out.key = error.key;
```

Comment:
Can you put // $FlowIssue above this line instead of casting the error to Object? That is more consistent with our intent.